etcd/restore-etcd-cluster-on-coreos.md

```diff
@@ -0,0 +1,110 @@
+# Restore etcd Cluster on CoreOS
+
+Here you will find step-by-step instructions on how to recover your etcd cluster. It is important to know that you can not restore your etcd cluster using only discovery URL. Discovery URL is used only once at bootstrap.
+
+In this example we will review 3-members etcd cluster with one failed node. It is important to take into consideration that this example supposes you used [Cloud-Config][cloud-config] with [discovery URL][etcd-discovery] to bootstrap your cluster with the following default options:
+
+```yaml
+  etcd2:
+    advertise-client-urls: http://%PEER_IP_ADDRESS%:2379
```

Comment:
`http://<PEER_ADDRESS>:2379` or `http://$PEER_ADDRESS:2379` (depending on the context, using a bash variable style like the second form may make the most sense).

Not `%PEER_ADDRESS%` as in this line -- let's not add a third variable form.